purple-pill/index.html

```diff
@@ -130,8 +130,9 @@
                     <div class="panel-border-without-menu">
                         <underconstruction>
                             <div class="bottom-text">
-                                Coming Soon:<br>Edit Metadata
+                                Coming Soon:<br>Edit Metadata<br><br>
```

Comment:
I did add the line-breaks to avoid overlapping the text with the progressbar. An alternative solution could be to make the progressbar thinner.